Source/Diagnostics/FullDiagnostics.cpp

```diff
@@ -700,11 +727,17 @@ FullDiagnostics::InitializeFieldFunctors (int lev)
             } else if ( m_varnames[comp] == "jy" ){
                 m_all_field_functors[lev][comp] = std::make_unique<JFunctor>(1, lev, m_crse_ratio, true, deposit_current);
                 deposit_current = false;
-            } else if ( m_varnames[comp] == "Ax" ){
+            } else if ( m_varnames[comp] == "jx_displacement" ){
+                m_all_field_functors[lev][comp] = std::make_unique<JdispFunctor>(0, lev, m_crse_ratio);
+            } else if ( m_varnames[comp] == "jy_displacement" ){
+                m_all_field_functors[lev][comp] = std::make_unique<JdispFunctor>(1, lev, m_crse_ratio);
+            }
+            else if ( m_varnames[comp] == "Ax" ){
```

Comment:
Just to stick with the convention used in the rest of this switch statement.
```suggestion
            } else if ( m_varnames[comp] == "Ax" ){
```